keyboards/playkbtw/helen80/helen80.c

```diff
@@ -16,5 +16,14 @@
 
 #include "helen80.h"
 
-// generated by KBFirmware JSON to QMK Parser
-// https://noroadsleft.github.io/kbf_qmk_converter/
\ No newline at end of file
+void keyboard_pre_init_kb(void) {
+  setPinOutput(B3);
+  setPinOutput(B2);
+}
+
+bool led_update_kb(led_t led_state) {
+  writePin(B3, !led_state.caps_lock);
+  writePin(B2, !led_state.scroll_lock);
+
+  return led_update_user(led_state);
+}
```

Comment:
Instead this should use <https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options>
```suggestion
```

And within `config.h`

```c
#define LED_CAPS_LOCK_PIN B3
#define LED_SCROLL_LOCK_PIN B2
#define LED_PIN_ON_STATE 0
```